synapse/rest/client/versions.py

```diff
@@ -44,7 +44,13 @@ def on_GET(self, request):
                     "r0.5.0",
                 ],
                 # as per MSC1497:
-                "unstable_features": {"m.lazy_load_members": True},
+                "unstable_features": {
+                    "m.lazy_load_members": True,
+                    # Advertise to clients that they need not include an `id_server`
+                    # parameter during registration of password reset, as Synapse now
```

Comment:
```suggestion
                    # parameter during registration or password reset, as Synapse now
```

Also don't forget adding a threepid which we still need to know we can do without the id_server aparam irrespective of binding